Management/src/ActuatorApi/ActuatorApi.http

```diff
@@ -0,0 +1,121 @@
+
+### DB Migrations actuator, list pending and applied database migrations
+GET {{ManagementAddress}}/{{Actuator_Context}}/dbmigrations
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Env actuator, list environment/configuration sources and data
+GET {{ManagementAddress}}/{{Actuator_Context}}/env
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Info actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/info
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Health actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/health
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Health actuator, check a specified group of contributors
+GET {{ManagementAddress}}/{{Actuator_Context}}/health/readiness
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Loggers actuator, list all loggers
+GET {{ManagementAddress}}/{{Actuator_Context}}/loggers
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Loggers actuator, update minimum level for a logger
+POST {{ManagementAddress}}/{{Actuator_Context}}/loggers/Steeltoe.Samples
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+{
+  "configuredLevel": "TRACE"
+}
+
+### Loggers actuator, reset minimum level for a logger  
+POST {{ManagementAddress}}/{{Actuator_Context}}/loggers/Steeltoe.Samples  
+Accept: application/json  
+Authorization: {{Authorization_Header_Value}}  
+{  
+  "configuredLevel": null  
+}
+
+### HTTP Exchanges actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/httpexchanges
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Mappings actuator, list registered HTTP routes
+GET {{ManagementAddress}}/{{Actuator_Context}}/mappings
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Metrics actuator, list all metrics
+GET {{ManagementAddress}}/{{Actuator_Context}}/metrics
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Metrics actuator, check a meter
+GET {{ManagementAddress}}/{{Actuator_Context}}/metrics/clr.process.uptime
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Prometheus endpoint, list all metrics
+GET {{ManagementAddress}}/{{Actuator_Context}}/prometheus
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Refresh actuator, refresh application configuration
+POST {{ManagementAddress}}/{{Actuator_Context}}/refresh
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Thread dump actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/threaddump
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
+
+### Heap dump actuator
+GET {{ManagementAddress}}/{{Actuator_Context}}/heapdump
+Accept: application/json
+Authorization: {{Authorization_Header_Value}}
```

Comment:
This one is failing (dhaka, linux cell, non-local publish):
> 502 Bad Gateway: Registered endpoint failed to handle the request.

Reply:
I just tried it and (after editing the timeout) got a 200, but Visual Studio seems to be struggling to render it. Maybe it'd be better to just remove from the .http file

Reply:
Yes, let's remove it.